cargo-workspaces/tests/publish.rs

```diff
@@ -0,0 +1,35 @@
+mod utils;
+use insta::assert_snapshot;
+
+#[test]
```

Comment:
I see some test failures caused by parallel execution. `#[serial]` should be added to all these tests.